docs/feature_encoders.md

```diff
@@ -158,6 +158,23 @@ bool encoder_update_user(uint8_t index, bool clockwise) {
 }
 ```
 
+Initialization code should be implemented in `encoder_init_state_kb`:
+
+```c
+void encoder_init_state_kb(uint8_t index) {
+    // set up encoder at the required index
+}
+
+To read an encoder that is not directly connected to an input pin but to another interface, implement `encoder_read_state_kb`:
+
+```c
+uint8_t encoder_read_state_kb(uint8_t index) {
+    return ... ; // This should return `-1`, `0`, or `1`, to match standard encoder "Gray Code" outputs.
+}
```

Comment:
I actually connected my encoder to the key matrix and try to get that working since I have no pins to spare, but it's not a use case for a `_delta` variant